examples/azure.yml

```diff
@@ -0,0 +1,101 @@
+kernel:
```

Comment:
could this YAML file be update to ressemble the other ones. The hashes don't match the latest and neither do the configs for the containers.